Documentation/api-reference.md

```diff
@@ -270,9 +275,9 @@ val qoeInfo = Media.createQoEObject(10000000L, 2D, 23D, 10D)
 
 Creates an instance of the Player State object.
 
-| Variable Name | Description | Required |
+| Parameter | Description | Required |
 | :--- | :--- | :---: |
-| `name` | State name\(Use Player State constants to track standard player states\) | Yes |
+| `name` | The player state name. Use Player State constants to track standard player states | Yes |
```

Comment:
As you state to use "Player State constants.." you should use the constants in the usage examples.